tests/test_otf.py

```diff
@@ -33,7 +33,7 @@ def cleanup(software="qe", casename="h2_otf_cp2k"):
     for f in glob.glob(f"*{software}.out"):
         os.remove(f)
 
-    if (software is 'qe'):
+    if (software == 'qe'):
```

Comment:
The parentheses are redundant :) (And Pep8 doesn't like that)

Reply:
ok. my autopep8 didn't manage to get rid of it...